sklearn/decomposition/dict_learning.py

```diff
@@ -832,7 +848,8 @@ def dict_learning_online(X, n_components=2, alpha=1, n_iter=100,
             print('|', end=' ')
         code = sparse_encode(X, dictionary.T, algorithm=method, alpha=alpha,
                              n_jobs=n_jobs, check_input=False,
-                             positive=positive_code)
+                             positive=positive_code, max_iter=method_max_iter,
+                             verbose=verbose)
```

Comment:
We may have avoided this because `sparse_encode` accepts a int for `verbose` and `verbose` in this context is a bool. (Although this will still work)

Reply:
We could pass `verbose > 0`, but I guess this way it's fine.